dist/constants.json

```diff
@@ -7825,697 +7825,697 @@
+        "label": "Lakeside VA Clinic"
       },
       {
         "value": "607GF",
-        "label": "FREEPORT CBOC"
+        "label": "Freeport VA Clinic"
       },
       {
         "value": "550GF",
-        "label": "MATTOON VA CLINIC"
+        "label": "Mattoon VA Clinic"
       },
       {
         "value": "578GC",
-        "label": "KANKAKEE COUNTY CBOC"
+        "label": "Kankakee County VA Clinic"
       },
       {
         "value": "578GE",
-        "label": "HOFFMAN ESTATES VA CLINIC"
+        "label": "Hoffman Estates VA Clinic"
       },
       {
         "value": "537",
-        "label": "JESSE BROWN VA MEDICAL CENTER"
+        "label": "Jesse Brown Department of Veterans Affairs Medical Center"
       },
       {
         "value": "550",
-        "label": "DANVILLE VA MEDICAL CENTER"
+        "label": "Danville VA Medical Center"
       },
       {
         "value": "556",
-        "label": "CAPTN JAMES LOVELL FED HLT CTR"
+        "label": "Captain James A. Lovell Federal Health Care Center"
       },
       {
         "value": "578",
-        "label": "EDWARD J. HINES JR. HOSPITAL"
+        "label": "Edward Hines Junior Hospital"
       }
     ],
     "WI": [
       {
         "value": "556GD",
-        "label": "KENOSHA CBOC"
+        "label": "Kenosha VA Clinic"
       },
       {
         "value": "585GB",
-        "label": "RHINELANDER CBOC"
+        "label": "Rhinelander VA Clinic"
       },
       {
         "value": "618BY",
-        "label": "TWIN PORTS CBOC"
+        "label": "Twin Ports VA Clinic"
       },
       {
         "value": "607GC",
-        "label": "JANESVILLE, WI CBOC"
+        "label": "Janesville VA Clinic"
       },
       {
         "value": "607GD",
-        "label": "BARABOO, WI CBOC"
+        "label": "Baraboo VA Clinic"
       },
       {
         "value": "607GE",
-        "label": "BEAVER DAM, WI CBOC"
+        "label": "Beaver Dam VA Clinic"
       },
       {
         "value": "618GE",
-        "label": "CHIPPEWA VALLEY CBOC"
+        "label": "Chippewa Valley VA Clinic"
       },
       {
         "value": "676GA",
-        "label": "WAUSAU CBOC"
+        "label": "Wausau VA Clinic"
       },
       {
         "value": "676GB",
         "label": "LEAVENWORTH, WI CBOC"
       },
       {
         "value": "676GC",
-        "label": "LACROSSE CBOC"
+        "label": "River Valley VA Clinic"
       },
       {
         "value": "695GD",
-        "label": "MILO C. HUEMPFNER CBOC"
+        "label": "Milo C. Huempfner Department of Veterans Affairs Outpatient Clinic"
       },
       {
         "value": "676GD",
-        "label": "WISCONSIN RAPIDS CBOC"
+        "label": "Wisconsin Rapids VA Clinic"
       },
       {
         "value": "676GE",
-        "label": "CLARK COUNTY CBOC"
+        "label": "Clark County VA Clinic"
       },
       {
         "value": "695BY",
-        "label": "FOX VALLEY"
+        "label": "John H. Bradley Department of Veterans Affairs Outpatient Clinic"
```

Comment:
- Consider abbreviated label for "Department of Veterans Affairs"
- Let's confirm this name change.

Reply:
The webpage for his says it's in the Fox Valley area and I don't see another facility named Fox Valley in this area. So that's good enough for me.

> proud to serve more than 11,000 Veterans in the Fox Valley area...